src/screens/ToggleGroup.js

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import { ToggleGroup } from 'grommet';
+import Page from '../components/Page';
+import Item from './Components/Item';
+import {
+  ComponentDoc,
+  Properties,
+  Property,
+  PropertyValue,
+  Description,
+  Example,
+  ThemeDoc,
+} from '../components/Doc';
+import { GenericPad, GenericBoolFalse } from '../utils/genericPropExamples';
+
+const ToggleGroupPage = () => (
+  <Page>
+    <ComponentDoc
+      name=""
+      availableAt={[
+        {
+          url: '',
+          badge:
+            'https://storybook.grommet.io/?path=/story/controls-togglegroup-simple--simple',
+          label: 'Storybook',
+        },
+        {
+          url: 'https://github.com/grommet/grommet/tree/master/src/js/components/ToggleGroup',
+          label: 'Github',
+        },
+      ]}
+      description="A collection of binary buttons capable of being switched between two states: on or off."
+      code={`<ToggleGroup
+  a11yTitle="Choose option"
+  options={['Option 1', 'Option 2', 'Option 3']}
+/>`}
+    >
+      <Properties>
+        <Property name="multiple">
+          <Description>
+            This will allow more than one of the buttons within ToggleGroup to
+            be selected.
+          </Description>
+          <GenericBoolFalse />
+        </Property>
+
+        <Property name="onToggle">
+          <Description>
+            Function that will be called when the user types or selects an
+            option in toggleGroup.
```

Comment:
```suggestion
            option.
```
Since this is ToggleGroup docs page I don't know that we need to say in ToggleGroup